.github/workflows/new-pages.yml

```
@@ -0,0 +1,41 @@
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)